fjord/suggest/providers/trigger/provider.py

```diff
@@ -37,6 +39,20 @@ def build_ga_category(rule):
     return 'trigger_{rule.slug}_{rule.id}'.format(rule=rule)
 
 
+def interpolate_url(url, response):
+    if '{' not in url:
+        return url
```

Comment:
You do this early-out optimization twice. Once here, and once in `TriggerRedirect.handle_redirect`.